src/config.c

```diff
@@ -2400,6 +2401,13 @@ static int isValidProcTitleTemplate(char *val, const char **err) {
     return 1;
 }
 
+/* Validate specified string is a valid locale string, meanwhile do
+the setting work. */
+static int isValidLocale(char *val, const char **err) {
+    const char *s = setlocale(LC_COLLATE, val);
```

Comment:
If s == NULL, err should be set to an appropriate error msg.

Reply:
Done